### PR TITLE
Only allow sceMpegGetAvcAu warmup for God Eater Series

### LINKED ISSUE #14086: FIFA 14 crash
#13999 sadly broke FIFA 14. Crashes right before the intro video plays.

### PR DESCRIPTION
**Pull request by sum2012**
Fix #14086

This is the worse thing that I find this fix God Eater.

### CODE REVIEW

File: Core/Compatibility.cpp [hunk 1]
```diff
@@ -79,6 +79,7 @@ void Compatibility::CheckSettings(IniFile &iniFile, const std::string &gameID) {
 	CheckSetting(iniFile, gameID, "ShaderColorBitmask", &flags_.ShaderColorBitmask);
 	CheckSetting(iniFile, gameID, "DisableFirstFrameReadback", &flags_.DisableFirstFrameReadback);
 	CheckSetting(iniFile, gameID, "DisableRangeCulling", &flags_.DisableRangeCulling);
+	CheckSetting(iniFile, gameID, "MpegWarmUpForGodEaterSeries", &flags_.MpegWarmUpForGodEaterSeries);
```
Comment (hrydgard) [inline Core/Compatibility.cpp:82]: Let's just call this MpegWarmUp. We might find other uses.

Comment (sum2012) [reply Core/Compatibility.cpp:82]: I will add warm up in  sceMpegGetAtracAu for fix another games , change MpegWarmUpForGodEaterSeries to  MpegWarmUp would confuse name.

Comment (hrydgard) [reply Core/Compatibility.cpp:82]: OK, for now. I hope eventually we can find a universal solution to these instead of relying on compat.ini hacks.

Comment (unknownbrackets) [reply Core/Compatibility.cpp:82]: I'm obviously not a fan of the idea of having an array of warm up flags for every HLE function.  If we have to do this though, MpegAvcWarmUp would be a better name.

I think we need to figure out what the actual variables are that cause the delay, which btw is seen in most pspautotests of Mpeg even with firmware 1.50 set.

-[Unknown]

Comment (hrydgard) [reply Core/Compatibility.cpp:82]: Renamed it.

Comment (sum2012) [reply Core/Compatibility.cpp:82]: @unknownbrackets My goal for sceMpegGetAtracAu 's warmup don't need add games in compat.ini

**Pull request merged**